peachjam/tasks.py

```diff
@@ -113,7 +113,9 @@ def run_ingestors():
 
     for ingestor in Ingestor.objects.all():
         if ingestor.enabled:
-            run_ingestor(ingestor.pk)
+            run_ingestor(
```

Comment:
this should be also be done when the ingestor object is saved (eg. via admin) -- I think it's reasonable to have a method on the ingestor that is called in the `save()` method, and can be called here too.

Reply:
Because we want the scheduled task to update when the ingestor is updated in the admin area.

Reply:
good idea, added

Reply:
to be picky: this method AND the the `.save()` method both run the ingestor in the same way -- logic is duplicated. Additionally, this method has to know how to use the internal details of the ingestor to know how to run it.

It's more object-oriented and better encapsulation to put the invocation logic into the Ingestor object. So `Ingestor.save()` calls `Ingestor.queue_task()`, and so does this. The Ingestor itself is responsible for knowing how to queue up its task given its state.

Reply:
Actually, it's done in THREE places now -- in the admin, too. There should be one method on the Ingestor that handles queuing up the task.